doc/dev/release_checklist.md

```diff
@@ -20,14 +22,24 @@ We'll use `v4.6.0` as the intended release version as a running example.
   - This step can take up to an hour.
   - If you are intending to cut the next release candidate on the same day,
     you may want to start on the release candidate checklist now.
+- Next we need to prepare the release notes.
+  - If the stable release is identical to the last release candidate (this should usually be the case),
+    you can reuse the release notes from `RELEASES.md`.
+  - If you want to regenerate the release notes,
+    use `script/release_notes.py --since v4.5.0`, run on the `releases/v4.6.0` branch,
+    and see the section "Writing the release notes" below for more information.
+  - Release notes should go in `RELEASES.md` on the `releases/v4.6.0` branch,
+    and should also be PR'd to `master` (suggested title: "chore: update release notes for v4.6.0").
 - Go to https://github.com/leanprover/lean4/releases and verify that the `v4.6.0` release appears.
-  - Edit the release notes on Github to select the "Set as the latest release".
-  - Follow the instructions in creating a release candidate for the "GitHub release notes" step,
-    now that we have a written `RELEASES.md` section.
-    Do a quick sanity check.
+  - Verify on Github that "Set as the latest release" is checked.
+  - Copy the generated release note into the text box, adding
```

Comment:
this sentence is incomplete. I suppose you wanted something like
```suggestion
  - Copy the generated release note into the text box, adding the header
    ```
    v4.6.0
    ----------
    ```
```